cmd/acr-credential-provider/main.go

```diff
@@ -70,7 +77,11 @@ func main() {
 		"Mirror a source registry host to a target registry host, and image pull credential will be requested to the target registry host when the image is from source registry host")
 
 	logs.AddFlags(command.Flags())
-	if err := command.Execute(); err != nil {
+	if err := func() error {
```

Comment:
I am interested in the coding style, why not simply
```
ctx, cancel := signal.NotifyContext(context.Background(), os.Interrupt)
defer cancel()
if err :=command.ExecuteContext(ctx); err != nil {
os.Exit(1)
}
```

Reply:
`os.Exit(1)` will skip the deferred call of `cancel()`

Reply:
In my understanding, os.Exist(1) should be called somewhere and it is here.

Reply:
Looks good, forget my last reply. I have removed my last comment for my misunderstanding the code.